README.md

```diff
@@ -1,36 +1,83 @@
-# n-gage
+# n-gage [![CircleCI](https://circleci.com/gh/Financial-Times/n-gage.svg?style=svg&circle-token=33bcf2eb98fe2e875cc66de93d7e4a50369c952d)](https://github.com/Financial-Times/n-gage)
 
-<a href="https://docs.google.com/forms/d/e/1FAIpQLSf5InA7UJK9yNBCzidFKI_WNkfbl6of1eRlIACRspGXUcBx8A/viewform?usp=pp_url&entry.78759464=n-gage" target="_blank"><img src="https://i.imgur.com/UmScdZ4.png" alt="Yak button" border="0" align="right" width="150" title="Report a yak shaving incident for this repository"></a>
+`n-gage` is in every FT.com project and gives a standard set of `make` tasks and the `ngage` CLI to help set up, build and deploy projects.
 
-`n-gage` is in every Next project, giving a standard set of `make` tasks and `ngage` CLI to help with setting up, building and deployments.
 
-<br clear="right">
+## Requirements
 
-## Getting started
+* Node 8.x
+* [Vault](https://github.com/Financial-Times/vault/wiki/Getting-Started-With-Vault)
 
-Starting a new repo?  You can do the following:
+
+## Installation
+
+```sh
+git clone git@github.com:Financial-Times/n-gage.git
+cd n-gage
+make install
+```
+
+## Development
+
+### Testing
+
+In order to run the tests locally you'll need to run:
+
+```sh
+make test
+```
+
+### Install from NPM
 
 ```sh
-mkdir my-new-project
-cd my-new-project
-npm init -y
-npm install --save-dev --no-package-lock @financial-times/n-gage
+npm install --save @financial-times/n-gage
```

Comment:
Based on [this discussion](https://financialtimes.slack.com/archives/GCWNMR4M8/p1596549573034000), it might be worth keeping `--no-package-lock`?